### ISSUE TITLE
Search filter makes no sense

### ISSUE THREAD

**Issue by thany** [mapsme/omim#11950]
I'm looking for a train station, so pressing the filter button in the search pane, I expected to select type "train station" or similar.

Instead, what I see, is checkin/checkout, rating, price, and (accommodation) type.

There are several problems with this:

1. These filters do not pertain to the search results shown, esp when no hotels are listed.
2. These filters do not allow narrowing search results for anything other than hotels, which is rarely sought after on a map like this (statistically speaking, because hotels comprise a tiny fraction of the total number of POI's).
3. The data for these filters is not in OSM, so filters like this go against the spirit of an offline OSM-based mapping tool.

If you (Maps.me devs) can show some solid data on how desirable these so-called filters are, please don't hesitate to post it right here. In layman's terms: who asked for this?
Otherwise I strongly suggest you remove these (for me useless) filters, and replace them with filters that:

1. Are applicable to the search results
2. Are more generic than just hotel filters
3. Draw from OSM data, not something proprietary or online

**Comment by burivuh** [comment]
These are hotel filters and are showed for hotels only. There's no such thing as a generic search filter in MAPS.ME.

**Comment by thany** [comment]
Then why is there a hotels filter when the search results contain no hotels?

And why only hotels? Can you please explain that? Maybe you can also reference the issue number where someone asked specifically for hotel filters *only*.

If not, then please consider implementing filters proper (you took this on yourselves by introducing these filters...) instead of brushing off this issue by closing it.

**Comment by burivuh** [comment]
You can create an issue on the search query that was falsely classified as a hotel search.

**Comment by thany** [comment]
I think you're missing the point.

There is a button named "Filters" which suggests generic filters. But in reality it only filters hotel metadata. That is what should be fixed. See my TS for how.
